Recipes/Recipes.Public/Recipe.cs

```diff
@@ -12,5 +12,5 @@ public class Recipe
     public required IEnumerable<string> Ingredients { get; init; }
     public required IEnumerable<string> Instructions { get; init; }
     public string? Image { get; init; }
-    public IEnumerable<Tag> Tags { get; init; } = new List<Tag>();
+    public IEnumerable<TagCreateUpdateDTO> Tags { get; init; } = new List<TagCreateUpdateDTO>();
```

Comment:
If i understand correctly, `Tags` should not be a list of `TagCreateUpdateDTO`, but rather a `TagDTO`.

Also `Recipe` is now inconsistent in terms of naming with `TagDTO`, but they are both types that we return